Makefile

```diff
@@ -21,7 +21,7 @@ generate: install-generator  ## Generate the SDK from our public openapi spec
 		-g python \
 		-o ./generated \
 		--additional-properties=packageName=groundlight_openapi_client
-	poetry run datamodel-codegen  --input spec/public-api.yaml --output generated/model.py
+	poetry run datamodel-codegen  --input spec/public-api.yaml --output generated/model.py --strict-nullable
```

Comment:
What does --strict-nullable do? For less obvious flags its helpful to comment them or link to their documentation

Reply:
Yeah, let me add a comment in the file itself, but the useful discussion is [here](https://github.com/koxudaxi/datamodel-code-generator/issues/327). For properties in the api spec that have `nullable: true` the generated pydantic class should let that property be Optional. For strange reasons, you need to set --strict-nullable for this to be true.